pkg/controllers/provisioning/suite_test.go

```diff
@@ -677,6 +677,39 @@ var _ = Describe("Volume Topology Requirements", func() {
 		}))[0]
 		ExpectNotScheduled(ctx, env.Client, pod)
 	})
+	It("should not relax an added volume topology zone node-selector away", func() {
+		persistentVolume := test.PersistentVolume(test.PersistentVolumeOptions{Zones: []string{"test-zone-3"}})
+		persistentVolumeClaim := test.PersistentVolumeClaim(test.PersistentVolumeClaimOptions{VolumeName: persistentVolume.Name, StorageClassName: &storageClass.Name})
+		ExpectApplied(ctx, env.Client, test.Provisioner(), storageClass, persistentVolumeClaim, persistentVolume)
+
+		pod := test.UnschedulablePod(test.PodOptions{
+			PersistentVolumeClaims: []string{persistentVolumeClaim.Name},
+			NodeRequirements: []v1.NodeSelectorRequirement{
+				{
+					Key:      "eks.amazonaws.com/capacityType",
```

Comment:
Can we use the karpenter label? Hate to proliferate a confusing label
```suggestion
					Key:      "v1alpha5.CapacityTypeLabel",
```

Reply:
Or is the intent that this is an invalid label?